checks/conciseness/conciseness.py

```diff
@@ -0,0 +1,84 @@
+from pydantic import ValidationError
+import json
+import openai
+
+from utils.check_schema import LLMToolkitStdCheckInputSchema, LLMToolkitStdCheckOutputSchema, ErrorSchema
+from prompt import compare_answers_prompt
+from utils.secret_manager import get_secret
+from utils.logger import get_logger
+
+
+logger = get_logger(__name__)
+def lambda_handler(event: dict, context: dict) -> dict:
+    logger.info(f"Going to run conciseness check on {event=}")
+    try:
+        body = LLMToolkitStdCheckInputSchema(**event)
+    except ValidationError  as ve:
+        response = ErrorSchema(
+            message="Bad Request Body",
+            reason=str(ve),
+        )
+        return {"statusCode": 400, "body": response.model_dump()}
+    
+    user_prompt, system_prompt = compare_answers_prompt(
+            question=body.question,
+            old_answer = body.old_answer,
+            new_answer=body.new_answer,
+            criterion="How concise the answer is"
+        )
+
+    try:
+        secrets = get_secret()
+    except Exception as e:
+        logger.error(f"Failed to retrieve secrets: {e}")
+        response = ErrorSchema(
+            message="Internal Server Error",
+            reason=str(e),
+        )
+        return {"statusCode": 500, "body": response.model_dump()}
+
+    try:
+        openai_api_key = secrets["OPENAI_API_KEY"]
+        openai.api_key = openai_api_key
+        openai_model = 'gpt-3.5-turbo'
+        response = openai.ChatCompletion.create(
+            model=openai_model,
+            messages=[
+                {'role': 'system', 'content': system_prompt},
+                {'role': 'user', 'content': user_prompt}
+            ],
+            temperature=0,
+        )
+        
+        check_result: str = response['choices'][0]['message']['content']
+    except Exception as e:
```

Comment:
Is there a known exception that openai.ChatCompletion.create returns ?
In that case we can replace the generic Exception with the known exception